applications/matter_weather_station/README.rst

```diff
@@ -204,26 +204,25 @@ To build this application to work with the nRF7002 Wi-Fi expansion board:
       .. group-tab:: nRF Connect for VS Code
 
          To build the application in the nRF Connect for VS Code IDE for Thingy:53 with the nRF7002 EB attached, add the expansion board and the build type variables in the build configuration's :guilabel:`Extra CMake arguments` and rebuild the build configuration.
-         For example: ``-- -DSHIELD=nrf7002eb -DFILE_SUFFIX=release``.
+         For example: ``-- -Dmatter_weather_station_SHIELD=nrf7002eb -DFILE_SUFFIX=release -DSB_CONFIG_WIFI_NRF700X=y``.
 
       .. group-tab:: Command line
 
          To build the sample from the command line for Thingy:53 with the nRF7002 EB attached, use the following command within the sample directory:
 
          .. code-block:: console
 
-            west build -b thingy53/nrf5340/cpuapp -- -DSHIELD=nrf7002eb -DFILE_SUFFIX=release
+            west build -b thingy53/nrf5340/cpuapp -- -Dmatter_weather_station_SHIELD=nrf7002eb -DFILE_SUFFIX=release -DSB_CONFIG_WIFI_NRF700X=y
 
 Generating factory data
 =======================
 
-To enable factory data support, you need to select the ``overlay-factory_data`` configuration overlay from the available application :ref:`build configuration overlays <matter_weather_station_app_build_configuration_overlays>`.
+To enable factory data support, you need to select the ``overlay-factory_data`` configuration overlay from the available application :ref:`build configuration overlays <matter_weather_station_app_build_configuration_overlays>`, set the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA` kconfig option to ``y``, and switch the file suffix to ``factory_data``.
```

Comment:
```suggestion
To enable factory data support, you need to select the ``overlay-factory_data`` configuration overlay from the available application :ref:`build configuration overlays <matter_weather_station_app_build_configuration_overlays>`, set the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA` kconfig option to ``y``, and use the ``factory_data`` file suffix.
```
I think this is correct, since a suffix isn't required by default so switching might not be the case.